core/edit/edit.talon

```diff
@@ -24,10 +24,11 @@ go down: edit.down()
 go line start: edit.line_start()
```

Comment:
head/tail fit a bit better here (which is where I have them defined…)

Reply:
better?